src/components/Calendar/Calendar.tsx

```diff
@@ -270,6 +273,8 @@ const Calendar: React.FC<CalendarProps> = ({
                                 meridiem: "short",
                             },
                             titleFormat: { year: "numeric", month: "short", day: "numeric" },
+                            calendarStartTime,
+                            calendarEndTime,
```

Comment:
are there any docs for these props? I'm struggling to find any

Reply:
slotMinTime/slotMaxTime are now being used instead to set the calendar hours.